bttc_rr_registration.html

```diff
@@ -0,0 +1,323 @@
+    }
+
+    button {
+      padding: 0.75rem;
+      background-color: #0d6efd;
+      color: white;
+      border: none;
+      border-radius: 8px;
+      font-size: 1rem;
+      cursor: pointer;
+    }
+
+    button:hover {
+      background-color: #0b5ed7;
+    }
+
+    .entry {
+      border: 1px solid #ddd;
+      background: #fafafa;
+      padding: 1rem;
+      border-radius: 8px;
+      margin-top: 1rem;
+      text-align: left;
+    }
+
+    .entry p {
+      margin: 0.5rem 0;
+    }
+
+    .result {
+      margin-top: 1.5rem;
+    }
+
+    .error {
+      color: #dc3545;
+      font-weight: bold;
+    }
+
+    .success {
+      color: #198754;
+      font-weight: bold;
+    }
+
+    .roster-link {
+      font-weight: bold;
+      color: #0d6efd;
+      text-decoration: underline;
+      margin-top: 0.5rem;
+      display: inline-block;
+    }
+
+    .token-field {
+      width: 70%;
+      margin-top: 0.5rem;
+    }
+
+    .confirm-btn {
+      margin-top: 0.5rem;
+    }
+
+    .token-error {
+      display: block;
+      margin-top: 0.5rem;
+      color: red;
+      font-size: 0.9em;
+    }
+  </style>
+</head>
+
+<body>
+
+  <div id="header">
+    <div class="container_12 clearfix">
+      <div id="logo" class="grid_6">
+        <a href="index.html" title="Go Home">
+          <img src="images/logo.gif" height="60" alt="BTTC Logo" />
+        </a>
+      </div>
+      <div id="search" class="grid_6"></div>
+    </div>
+    <div class="border">
+      <ul id="nav" class="container_12 clearfix">
+        <li class="grid_2"><a href="index.html" title="Home">Home</a></li>
+        <li class="grid_2"><a href="results.html" title="Weekly Results">Results</a></li>
+        <li class="grid_2"><a href="coaching.html" title="Coaching">Coaching</a></li>
+        <li class="grid_2"><a href="about.html" title="About Us">About Us</a></li>
+        <li class="grid_2"><a href="club_policy.html" title="Club Policy">Club Policy</a></li>
+        <li class="grid_2 last"><a href="more.html" title="More...">More...</a></li>
+      </ul>
+    </div>
+  </div>
+
+  <div class="container" id="lookupContent">
+    <h3>Round Robin Sign-Up</h3>
+    <p>Enter your phone number (e.g., 5101234567):</p>
+    <form id="lookupForm">
+      <input type="text" id="phoneInput" name="phone" placeholder="Enter phone number" required />
+      <button type="submit">Lookup</button>
+    </form>
+    <div class="result" id="result"></div>
+    <div class="result" id="registrationForm"></div>
+    <a href="roster.html" class="roster-link">
+      <i class="fas fa-users"></i> VIEW ROSTER
+    </a>
+  </div>
+
+  <script>
+    const scriptUrl = 'https://script.google.com/macros/s/AKfycbzhP9_u4ShN6C8wd7Ro_XR9VukeCI-lPyejCmB-Nn8YbdNNSexeZ0bsETktsR_P8RS-/exec';
+    let players = [];
+
+    document.getElementById('lookupForm').addEventListener('submit', function (e) {
+      e.preventDefault();
+      const phone = document.getElementById('phoneInput').value.trim();
```

Comment:
I suggest adding a little regex here to strip out any non-digit characters. Also removes the number `1` if they enter their phone number starting with `1`. This would be handy to clean up auto-fill and this way people can enter their phone number in any format they like.

![Screen Shot 2025-05-31 at 3 56 34 PM](https://github.com/user-attachments/assets/96716e53-8374-45bb-b6a2-c5a4bc732190)

```suggestion
      const phone = document.getElementById('phoneInput').replace(/^1|\D/g, '');
```